server/app/views/admin/programs/ManageProgramAdminsView.java

```diff
@@ -64,72 +71,105 @@ public Content render(
             .setTitle(fullTitle)
             .addMainContent(
                 renderHeader(fullTitle),
-                adminEmailTemplate(),
-                renderAdminForm(request, program.id(), existingAdminEmails));
+                renderHeader("Add New Admin"),
+                renderAddNewButton(request, program.id()),
+                renderHeader("Existing Admins"),
+                renderExistingAdmins(request, program.id(), existingAdminEmails),
+                renderBackButton());
```

Comment:
In most of our other screens, we have the back button at the top of the page

Reply:
Done